package.json

```diff
@@ -75,6 +75,7 @@
     "cat-names": "^2.0.0",
     "compatibility": "^1.0.0",
     "dog-names": "~1.0.2",
+    "funding": "^1.0.3",
```

Comment:
Is it clear that Feross as an individual has squatted on the "funding" name in the npm global namespace to collect (and distribute?) all of the proceeds raised through distributing this adware?